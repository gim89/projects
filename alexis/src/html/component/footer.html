<footer class="footer-page">
  <div class="container footer-page__container">
    <div class="section__body footer-page__body">
      <a href="#" class="footer-page__logo">Alexis</a>
      <div class="footer-page__item">
        <h3 class="footer-page__title color__green">
          Alexis LLC.
        </h3>
        <p class="footer-page__text">
          4125 Brighton Circle Rood
        </p>
        <p class="footer-page__text">
          Little Falls, MN 56345
        </p>
      </div>
      <div class="footer-page__item">
        <h3 class="footer-page__title color__purple">
          Alexis LLC UK.
        </h3>
        <p class="footer-page__text">
          81 Golden Knowes Road
        </p>
        <p class="footer-page__text">
          Freshfield l37 6tp
        </p>
      </div>
      <div class="footer-page__item">
        <h3 class="footer-page__title color__orange">
          Follow Us.
        </h3>
        <div class="footer-page__social">
          <a href="#" class="footer-page__icons footer-page__facebook"></a>
          <a href="#" class="footer-page__icons footer-page__twitter"></a>
          <a href="#" class="footer-page__icons footer-page__instagram"></a>
          <a href="#" class="footer-page__icons footer-page__google"></a>
        </div>
      </div>
    </div>
    <div class="footer-page__lines">
      <div class="footer-page__line line background__red"></div>
      <div class="footer-page__line line background__green"></div>
      <div class="footer-page__line line background__purple"></div>
      <div class="footer-page__line line background__orange"></div>
    </div>
  </div>
</footer>