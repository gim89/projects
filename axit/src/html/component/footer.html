<footer class="footer-page">
  <div class="footer-page__container container">
    <div class="footer-page__social">
      <a href="#" class="footer-page__icon footer-page__facebook footer-page__facebook--hover"></a>
      <a href="#" class="footer-page__icon footer-page__twitter footer-page__twitter--hover"></a>
      <a href="#" class="footer-page__icon footer-page__google footer-page__google--hover"></a>
      <a href="#" class="footer-page__icon footer-page__pinterest footer-page__pinterest--hover"></a>
      <a href="#" class="footer-page__icon footer-page__instagram footer-page__instagram--hover"></a>
      <a href="#" class="footer-page__icon footer-page__stumbleupon footer-page__stumbleupon--hover"></a>
      <a href="#" class="footer-page__icon footer-page__rss footer-page__rss--hover"></a>
    </div>
    <p class="footer-page__text">&copy; 2015 Axure Themes</p>
  </div>
</footer>