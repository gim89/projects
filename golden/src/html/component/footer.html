<footer class="footer-page">
  <div class="container footer-page__container">
    <p class="footer-page__text">&copy; Copyright 2014 FreebiesXpress.com</p>
    <div class="footer-page__social">
        <a href="#" class="social__icon twitter twitter--hover footer-page__icon"></a>
        <a href="#" class="social__icon facebook facebook--hover footer-page__icon"></a>
        <a href="#" class="social__icon pinterest pinterest--hover footer-page__icon"></a>
        <a href="#" class="social__icon google google--hover footer-page__icon"></a>
    </div>
  </div>
</footer>